### PR TITLE
add user.intercom_hash

### PR DESCRIPTION
**Pull request by jperl**

### CODE REVIEW

File: app/graphql/queries.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {
 export const currentUserQuery = gql`
   query currentUser {
     currentUser {
+      intercom_hash
```
Comment (flaurida) [inline app/graphql/queries.ts:24]: why not add to `UserFragment`?

Comment (jperl) [reply app/graphql/queries.ts:24]: It's not actually included everywhere only in CurrentUser query

**Review by flaurida — APPROVED** [review]
1 minor comment then lgtm

File: app/graphql/queries.ts (hunk 1 above)
Comment (flaurida) [reply app/graphql/queries.ts:24]: ok

**Pull request merged**